### PR TITLE
feat:Added a confirm password to sign up

### LINKED ISSUE #797: Adding Confirm Password in SignUp
**Is it a ?**

- [ ] Bug Report

- [x] Feature Request
    
- [ ] Chore
    
### Feature Request

**Feature Description**
There should be a confirm password screen in the sign up section of the app

**Description**
Currently there is no confirm password section.After a password is entered , there should be a confirm password edit text as well to check whether both of them match.

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #797 
A confirm password edittext was added to Sign Up section and the functionality was implemented.

Screenshots for the change: 
whatsapp image 2018-03-20 at 11 07 13

### CODE REVIEW

**Comment by sridharjajoo** [comment]
@iamareebjamal Kindly review!Thanks

**Comment by iamareebjamal** [comment]
@sridharjajoo Please fix the build first

File: app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java [hunk 1]
```diff
@@ -66,7 +66,10 @@ public void onStart() {
         binding.btnSignUp.setOnClickListener(view -> {
             if (!validator.validate())
                 return;
-
+            if (!((binding.password.getText().toString())).equals(binding.confirmPassword.getText().toString())){
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java]: Use presenter to do validation

File: app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java [hunk 2]
```diff
@@ -68,6 +68,12 @@ public void onStart() {
                 return;
 
             String url = binding.url.baseUrl.getText().toString().trim();
+            String password = binding.password.getText().toString();
+            String confirmPassword = binding.confirmPassword.getText().toString();
+            if (!(getPresenter().comparePassword(password, confirmPassword))) {
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java]: AFAIK, presenter is already saved in a field?

Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java]: This translates to `if passwords are not compared, return` 

Functions with verbs as names return void, so this doesn't make sense

Functions which return boolean should be named as `arePasswordsEqual`

Comment (sridharjajoo) [reply app/src/main/java/org/fossasia/openevent/app/module/auth/signup/SignUpFragment.java]: I apologize . I'll do the required changes.

**Comment by sridharjajoo** [comment]
@iamareebjamal I am having an error pmd file not found

**Comment by iamareebjamal** [comment]
You have PMD errors. check the generated html file to check what the errors are

**Comment by sridharjajoo** [comment]
" Task 'pmd' not found in root project 'open-event-orga-app'  " . This is the error shown on running it from cmd.

**Comment by iamareebjamal** [comment]
You have to run `./gradlew androidCheck`

**Comment by sridharjajoo** [comment]
@iamareebjamal The build is passing now :)

**Pull request merged**